test/functional/test_site/index.md

```diff
@@ -133,6 +133,10 @@ tags: ["tag-frontmatter-shown", "tag-included-file", "+tag-exp*", "-tag-exp-hidd
 
 ## <include src="testTrimInclude.md" trim inline />
 
+# Trimmed include fragment
+
+## Before <include src="testTrimIncludeFragment.md#fragment" trim inline /> After
```

Comment:
Could we make the Before and After parts more visually distinct? For example:
```
Before | <include src="testTrimIncludeFragment.md#fragment" trim inline /> | After
```
or `Before` and `After` could be in a different colour?